gluten-substrait/src/main/scala/org/apache/gluten/planner/cost/RoughCostModel.scala

```diff
@@ -16,50 +16,62 @@
  */
 package org.apache.gluten.planner.cost
 
-import org.apache.gluten.execution.RowToColumnarExecBase
+import org.apache.gluten.GlutenConfig
 import org.apache.gluten.extension.columnar.transition.{ColumnarToRowLike, RowToColumnarLike}
 import org.apache.gluten.utils.PlanUtil
 
 import org.apache.spark.sql.catalyst.expressions.{Alias, Attribute, NamedExpression}
-import org.apache.spark.sql.execution.{ColumnarToRowExec, ProjectExec, RowToColumnarExec, SparkPlan}
-import org.apache.spark.sql.types.{ArrayType, MapType, StructType}
+import org.apache.spark.sql.execution.{ColumnarToRowExec, DataSourceScanExec, LeafExecNode, ProjectExec, RowToColumnarExec, SparkPlan}
+import org.apache.spark.sql.execution.adaptive.AdaptiveSparkPlanExec
 
 /** A rough cost model with some empirical heuristics. */
 class RoughCostModel extends LongCostModel {
 
+  private def getSizeFactor(plan: SparkPlan): Long = {
+    // Get the bytes size that the plan needs to consume
+    val sizeBytes = plan match {
+      case scan: DataSourceScanExec => getStatSizeBytes(scan)
+      case _: LeafExecNode => 0L
```

Comment:
Why this line is needed?

Reply:
Some LeafExecNode bytesSize is particularly large and inaccurate, such as ExecutedCommandExec, which is ignored here. In addition, the AdaptiveSparkPlanExec match is lost here. I will fix it immediately.